aasemble/django/apps/mirrorsvc/models.py

```diff
@@ -125,6 +125,7 @@ class Snapshot(models.Model):
     uuid = models.UUIDField(unique=True, default=uuid.uuid4, editable=False)
     timestamp = models.DateTimeField(auto_now_add=True)
     mirrorset = models.ForeignKey(MirrorSet)
+    tag = models.CharField(max_length=200)
```

Comment:
I think it's better if we make it a separate model with a ForeignKey back to Snapshot. That way, we can more efficiently search for Snapshots that have a given tag.